src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -245,4 +249,27 @@ export class NotebookMarkdownRenderer {
 	setNotebookURI(val: URI) {
 		this._notebookURI = val;
 	}
+
+	/**
+	 * The following is a sample cell attachment from JSON:
+	 *  "attachments": {
+	 *     "test.png": {
+	 *        "image/png": "iVBORw0KGgoAAAANggg==="
+	 *     }
+	 *  }
+	 */
+	findAttachmentIfExists(href: string, cellAttachments: { [key: string]: { [key: string]: string } }): string {
```

Comment:
This doesn't really need to be a class member - you can just have it be a module level function